src/qtiCommonRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -124,13 +122,8 @@ var render = function render(interaction) {
                         // declare the item ready when player is ready to play.
                         resolve();
                     })
-                    .on(
-                        'update',
-                        _.throttle(function() {
-                            containerHelper.triggerResponseChangeEvent(interaction);
-                        }, 1000)
-                    )
-                    .on('ended', function() {
+                    .on('update', _.throttle(() => containerHelper.triggerResponseChangeEvent(interaction), 1000))
+                    .on('ended', function () {
```

Comment:
Why did you keep the anonymous function here?

Reply:
The lexical `this` is needed to be on the mediaplayer component.